addon/components/frost-list-item-content.js

```diff
@@ -0,0 +1,103 @@
+import Ember from 'ember'
+const {get, isPresent} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {Component} from 'ember-frost-core'
+import {PropTypes} from 'ember-prop-types'
+
+import layout from '../templates/components/frost-list-item-content'
+
+export default Component.extend({
+
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+  layout,
+
+  // == PropTypes =============================================================
+
+  propTypes: {
+    // Options - required
+    model: PropTypes.EmberObject.isRequired,
+    index: PropTypes.number.isRequired,
+    onExpand: PropTypes.func.isRequired,
+    onSelect: PropTypes.func.isRequired,
+
+    // Options - general
+    onSelectionChange: PropTypes.func,
+    itemTypes: PropTypes.object,
+    itemTypeKey: PropTypes.string,
+    isAnyTypedItemExpansion: PropTypes.bool,
```

Comment:
What is this property doing?

Reply:
It's used to determine if there are any typed item expansion components present. If this is the case, there is logic to render a placeholder `<div>` where the expansion icon would normally be, so that any list items without expansion are aligned with list items that have expansion. This was not an issue previously because either all of the list items would have expansion, or none of them would. Other cases where this computed property is used are to conditionally render certain elements, such as the collapse all/expand all controls at the top of the list.